google-cloud-logging/lib/google/cloud/logging/project.rb

```diff
@@ -266,17 +270,20 @@ def entry log_name: nil, resource: nil, timestamp: nil, severity: nil,
         #   resource = logging.resource "gae_app",
         #                               "module_id" => "1",
         #                               "version_id" => "20150925t173233"
+        #   partial_success = true
         #
         #   logging.write_entries [entry1, entry2],
         #                         log_name: "my_app_log",
         #                         resource: resource,
-        #                         labels: labels
+        #                         labels: labels,
+        #                         partial_success: true
```

Comment:
I think you wanted `partial_success: partial_success` to match the corresponding samples below. Otherwise the line `partial_success = true` above is redundant.

Reply:
Ah, yes. Thanks for catching this. I've changed it.